pkg/executor/join/base_join_probe.go

```diff
@@ -65,7 +70,8 @@ type ProbeV2 interface {
 	// NeedScanRowTable returns true if current join need to scan row table after all the probe side chunks are handled
 	NeedScanRowTable() bool
 	// InitForScanRowTable do some pre-work before ScanRowTable, it must be called before ScanRowTable
-	InitForScanRowTable()
+	InitForScanRowTable(inSpillMode bool)
+	GetSpilledRowNum() map[int]int // TODO remove it
```

Comment:
will it be removed in this pr or later?

Reply:
> will it be removed in this pr or later?

Yes, we will remove it before this pr is merged.